lib/fog/compute/google/requests/insert_instance_group_manager.rb

```diff
@@ -0,0 +1,29 @@
+module Fog
+  module Compute
+    class Google
+      class Mock
+        def insert_instance_group_manager(_name, _zone, _instance_template, _base_instance_name,
+                                          _target_size, _target_pools, _named_ports, _description)
+          Fog::Mock.not_implemented
+        end
+      end
+
+      class Real
+        def insert_instance_group_manager(name, zone, instance_template, base_instance_name,
+                                          target_size, target_pools, named_ports, description)
+          instance_group_manager = ::Google::Apis::ComputeV1::InstanceGroupManager.new(
+            :description => description,
+            :name => name,
+            :instance_template => instance_template.self_link,
```

Comment:
Nit: This assumes that an object is going to be passed in.
Usually models should accept self_links and resource names as well.

Reply:
Done in the three requests.